.github/workflows/ci.yml

```diff
@@ -0,0 +1,96 @@
+name: testing
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches:
+      - master
+      - lispci/master
```

Comment:
Can you please remove "lispci/master" ?

Reply:
Done.

One thing I'm wondering is whether we should add `workflow_dispatch` for easier debugging.  My commit with this fix will uncomment that line, but let me know if you don't want it.